File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TableViews_View.cpp

#include "TableViews_View.h"
#include "TableViews_View.h"
#include <vector>
#include <memory>
#include "TableViews_ViewMyElementsRow.h"
#include "TableViews_ViewMyStringRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  std::vector<std::shared_ptr<TableViews_ViewMyElementsRow>> TableViews_View::getMyElementsTableRows( ) 
  {
    return this->MyElementsTableRows;
  }
  int TableViews_View::getMyElementsTableSelectedRow( ) 
  {
    return this->MyElementsTableSelectedRow;
  }
  bool TableViews_View::getIsMyElementsTableVisible( ) 
  {
    return this->isMyElementsTableVisible;
  }
  bool TableViews_View::getIsMyElementsTableEnabled( ) 
  {
    return this->isMyElementsTableEnabled;
  }
  bool TableViews_View::getIsMyElementsTableDescriptionColumnVisible( ) 
  {
    return this->isMyElementsTableDescriptionColumnVisible;
  }
  std::vector<std::shared_ptr<TableViews_ViewMyStringRowHandlesRow>> TableViews_View::getMyStringRowHandlesTableRows( ) 
  {
    return this->MyStringRowHandlesTableRows;
  }
  std::string TableViews_View::getMyStringRowHandlesTableSelectedRow( ) 
  {
    return this->MyStringRowHandlesTableSelectedRow;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TableViews_View.cs
/// <filename>
///     TableViews_View.cs
/// </filename>
using System.Collections.Generic;
using widgetassertions;

namespace widgetassertions
{
    public class TableViews_View
  {
        private System.Collections.Generic.List<widgetassertions.TableViews_ViewMyElementsRow> MyElementsTableRows = new System.Collections.Generic.List<widgetassertions.TableViews_ViewMyElementsRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TableViews_ViewMyElementsRow> getMyElementsTableRows()
    {
      return this.MyElementsTableRows;
    }
     
        private int MyElementsTableSelectedRow;
     
        public     virtual int getMyElementsTableSelectedRow()
    {
      return this.MyElementsTableSelectedRow;
    }
     
        private bool isMyElementsTableVisible;
     
        public     virtual bool getIsMyElementsTableVisible()
    {
      return this.isMyElementsTableVisible;
    }
     
        private bool isMyElementsTableEnabled;
     
        public     virtual bool getIsMyElementsTableEnabled()
    {
      return this.isMyElementsTableEnabled;
    }
     
        private bool isMyElementsTableDescriptionColumnVisible;
     
        public     virtual bool getIsMyElementsTableDescriptionColumnVisible()
    {
      return this.isMyElementsTableDescriptionColumnVisible;
    }
     
        private System.Collections.Generic.List<widgetassertions.TableViews_ViewMyStringRowHandlesRow> MyStringRowHandlesTableRows = new System.Collections.Generic.List<widgetassertions.TableViews_ViewMyStringRowHandlesRow>();
     
        public     virtual System.Collections.Generic.List<widgetassertions.TableViews_ViewMyStringRowHandlesRow> getMyStringRowHandlesTableRows()
    {
      return this.MyStringRowHandlesTableRows;
    }
     
        private string MyStringRowHandlesTableSelectedRow;
     
        public     virtual string getMyStringRowHandlesTableSelectedRow()
    {
      return this.MyStringRowHandlesTableSelectedRow;
    }
  }
}

